deployment-aws/terraform/globus.tf

```diff
@@ -0,0 +1,114 @@
+data "aws_s3_object" "deployment_key" {
+  bucket = "webmev-tf"
+  key    = "secrets/${local.stack}/deployment-key.${local.stack}.json"
```

Comment:
Would `secrets/${local.stack}/deployment-key.json` be enough to disambiguate?

Reply:
I did that to keep the deployment keys (and node config JSON) clear on my local machine. However, I can simplify since it's excessive there. Will fix.